From e352738817710afd6e6dbf594120cccafa387c18 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Sun, 15 Nov 2020 17:16:26 +0200
Subject: [PATCH] [SW-0] fwupd: reorganize the includes

Move the generic includes to the top for better structured
includes list.

Change-Id: Ic9e819c7436d775a6d1798547b44139efb3f8568
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/preboot/module_fwupd.c b/preboot/module_fwupd.c
index 523fd94..ac447f5 100644
--- a/preboot/module_fwupd.c
+++ b/preboot/module_fwupd.c
@@ -4,13 +4,14 @@
  * All Rights Reserved.
  */
 
+#include <stdint.h>
+#include <inttypes.h>
+#include <string.h>
+
 #include <psoc_global_conf_regs.h>
 #include <hl_fw_if.h>
 #include <preboot.h>
-#include <string.h>
 #include <fit_load_type.h>
-#include <stdint.h>
-#include <inttypes.h>
 
 #include "cmd_module.h"
 #include "uart.h"
